katdal/__init__.py

```diff
@@ -240,7 +242,9 @@
 _dataset, _concatdata, _sensordata = dataset, concatdata, sensordata
 _h5datav1, _h5datav2, _h5datav3 = h5datav1, h5datav2, h5datav3
 _categorical, _lazy_indexer = categorical, lazy_indexer
-del dataset, concatdata, h5datav1, h5datav2, h5datav3, sensordata, categorical, lazy_indexer
+_spectral_window, _visdatav4 = spectral_window, visdatav4
+del dataset, concatdata, sensordata, h5datav1, h5datav2, h5datav3
+del categorical, lazy_indexer, spectral_window, visdatav4
```

Comment:
Can we just get rid of these? It leads to some bizarre things (see my [PyCon lightning talk](http://www.brucemerry.org.za/pycon2018/imports.slides.html#/2).

Reply:
Indeed, I've always wanted to... I'll make a new PR that removes the `del`s and rather put the public API in an `__all__` list.